### ISSUE TITLE
Appdaemon not working on newest hass.io install

### ISSUE THREAD

**Issue by Vexilyze** [hassio-addons/addon-appdaemon3#42]
# Problem/Motivation

Since the recent update in hass.io appdaemon does not seem to want to start.

## Expected behavior

I even tried to uninstall it completely to do a fresh install of appdaemon, yet still with a fresh install it doesnt want to start.

## Actual behavior

```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-banner.sh: executing... 
-----------------------------------------------------------
 Hass.io Add-on: AppDaemon
 Python Apps and HADashboard using AppDaemon 3.x for Home Assistant
-----------------------------------------------------------
 Add-on version: 2.0.2
 You are running the latest version of this add-on.
 System: HassOS 1.13  (aarch64 / raspberrypi3-64)
 Home Assistant version: 0.91.0
 Supervisor version: 152
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in e.g, GitHub, forums or the Discord chat.
-----------------------------------------------------------
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] 20-init-configuration.sh: executing... 
[cont-init.d] 20-init-configuration.sh: exited 0.
[cont-init.d] 21-compiled-dir.sh: executing... 
[cont-init.d] 21-compiled-dir.sh: exited 0.
[cont-init.d] 30-auto-token.sh: executing... 
[14:21:30] INFO: Updating Hass.io API token in AppDaemon with the current one...
[cont-init.d] 30-auto-token.sh: exited 0.
[cont-init.d] 31-ha-url.sh: executing... 
[cont-init.d] 31-ha-url.sh: exited 0.
[cont-init.d] 50-compiled-symlink.sh: executing... 
[cont-init.d] 50-compiled-symlink.sh: exited 0.
[cont-init.d] 80-system-packages.sh: executing... 
[cont-init.d] 80-system-packages.sh: exited 0.
[cont-init.d] 81-python-packages.sh: executing... 
[cont-init.d] 81-python-packages.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[14:21:31] INFO: Starting AppDaemon...
[14:21:32] INFO: Starting AppDaemon...
[14:21:33] INFO: Starting AppDaemon...
[14:21:34] INFO: Starting AppDaemon...
[14:21:35] INFO: Starting AppDaemon...
[14:21:36] INFO: Starting AppDaemon...
```

## Steps to reproduce

Simply update everything to the newest version and you probably will get the same issue.

## Proposed changes

> (If you have a proposed change, workaround or fix,
> describe the rationale behind it)

**Comment by MightyMilster** [comment]
I had the same problem. I **suspect** _(I am not an expert by any means)_ the reason is that v2.0.2 is not currently playing nice with 64bit builds. I rebuilt my installation on 32bit ([HA recommended](https://www.home-assistant.io/hassio/installation/)) and I can get it to run fine now.

You have this in your log...
`System: HassOS 1.13  (aarch64 / raspberrypi3-64)`

**Comment by frenck** [comment]
Yes, is caused by the 64 bits (aarch64), see here for the upstream issues:

- gliderlabs/docker-alpine#486
- https://bugs.alpinelinux.org/issues/9981
- https://bugs.python.org/issue36107

I'm currently in the process of replacing all add-ons that rely on Python to a different base image with Python 3.7.3 in a similar fashion as Home Assistant has done.

**Comment by frenck** [comment]
This issue has been solved in the v3.0.0 release of this add-on.
